src/backend/utils/misc/guc.c

```diff
@@ -2668,6 +2669,17 @@ static struct config_int ConfigureNamesInt[] =
 		NULL, NULL, NULL
 	},
 
+	{
+		{"stream_insertion_commit_interval", PGC_BACKEND, QUERY_TUNING_OTHER,
+		 gettext_noop("Sets the default amount of time to periodically commit from a stream insertion process."),
+		 gettext_noop("A lower number will minimize the amount of time the autovacuumer can be blocked from freeing unused space by long-running stream insertion processes."),
```

Comment:
Maybe we should also add that a lower number will make sure newly created CVs are seen by the insert process faster, otherwise if a CV is created in the middle of a long running INSERT transaction, it will not see any data.